py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -1289,3 +1289,22 @@ def aten_ops_convolution(
         dilation=args[5],
         groups=args[8],
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.linear)
```

Comment:
This just needs to be renamed to `torch.ops.aten.linear.default`, then it is good to go.

Reply:
@gs-olive Thanks for your reminder! However, `linear` op seems not like other ops to load `default`. It just looks for `torch.ops.aten.linear`. See pic below:
![image](https://github.com/pytorch/TensorRT/assets/29559374/44894ee7-c0ba-4924-815c-ef81850cc0df)

I was wondering how we can tell what ops need `default` or not (even though almost all ops need `default` so far, to my knowledge)

Reply:
Ah I see - thanks for that. I believe the current behavior is that all operators have a `default` overload, but when they are not specified with a `default` overload, they point to the `default` behavior. So, `torch.ops.aten.linear` as a callable is the same as `torch.ops.aten.linear.default`. It should be okay to check for ``torch.ops.aten.linear` in the test case, but use the overload of `default` in the converter, as you see fit, since those would be interchangeable in the graph.

Reply:
Ahh I see. You're right. I just tried to use `@dynamo_tensorrt_converter(torch.ops.aten.linear.default)` decorator and run the test with `expected_ops={torch.ops.aten.linear}`. It still can pass. Thanks for the explanation!